allensdk/test/api/cloud_cache/test_cache.py

```diff
@@ -746,3 +727,92 @@ def test_latest_manifest_warning(tmpdir, example_datasets_with_metadata):
     assert 'It is possible that some data files' in msg
     cmd = "S3CloudCache.load_manifest('project-x_manifest_v4.0.0.json')"
     assert cmd in msg
+
+
+@mock_s3
+def test_load_last_manifest(tmpdir, example_datasets_with_metadata):
+    """
+    Test that load_last_manifest works
+    """
+    bucket_name = 'load_lst_manifest_bucket'
+    metadatasets = example_datasets_with_metadata['metadata']
+    create_bucket(bucket_name,
+                  example_datasets_with_metadata['data'],
+                  metadatasets=metadatasets)
+
+    cache_dir = pathlib.Path(tmpdir) / 'load_last_cache'
+    cache = S3CloudCache(cache_dir, bucket_name, 'project-x')
+
+    # check that load_last_manifest in a new cache loads the
+    # latest manifest without emitting a warning
+    with pytest.warns(None) as warnings:
+        cache.load_last_manifest()
+    ct = 0
+    for w in warnings.list:
```

Comment:
doesn't `pytest.warns(None)` already assert that there are no warnings?

Reply:
Annoyingly, no

https://docs.pytest.org/en/stable/warnings.html#recwarn